pgtrigger/apps.py

```diff
@@ -6,7 +6,7 @@
 def install(**kwargs):
     import pgtrigger
 
-    pgtrigger.install()
+    pgtrigger.install(**kwargs)
```

Comment:
This change doesn't accomplish anything from what I can tell. This is only called after the post_migrate hook when configured to ensure triggers are installed after migrations. Can you revert?